.config/nextest.toml

```diff
@@ -38,6 +38,6 @@ slow-timeout = { period = "120s", terminate-after = 3 }
 # There might be some network issues, so we allow some retries with backoff.
 # Jitter is enabled to avoid [thundering herd issues](https://en.wikipedia.org/wiki/Thundering_herd_problem).
 [[profile.default.overrides]]
-filter = 'test(rpc_test_)'
+filter = 'test(rpc_snapshot_test_)'
 slow-timeout = { period = "120s", terminate-after = 3 }
 retries = { backoff = "exponential", count = 4, delay = "5s", jitter = true }
```

Comment:
If this mitigates the flakiness, then the `retries` should not be needed anymore.